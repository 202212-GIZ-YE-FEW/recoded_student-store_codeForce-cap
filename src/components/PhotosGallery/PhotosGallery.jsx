import { SectionWrapper } from "@/hoc"
import React from "react"
import Image from "next/image"

function PhotosGallery() {
  return (
    <div>
      <div>
        <Image
          className='h-full w-full max-w-full object-cover'
          src='photo-one.svg'
          width={500}
          height={500}
          alt='Photo-one'
        />
      </div>
      <div>
        <Image
          className='h-full w-full object-cover'
          src='photo-two.svg'
          width={100}
          height={100}
          alt='photo-two'
        />
        <Image
          className='h-full w-full object-cover'
          src='photo-three.svg'
          width={100}
          height={100}
          alt='photo-three'
        />
        <Image
          className='h-full w-full object-cover'
          src='photo-four.svg'
          width={100}
          height={100}
          alt='photo-four'
        />
      </div>
    </div>
  )
}

export default SectionWrapper(PhotosGallery)
