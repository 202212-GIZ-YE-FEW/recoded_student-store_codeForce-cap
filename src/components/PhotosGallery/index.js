import PhotosGallery from "../PhotosGallery/PhotosGallery"
export default PhotosGallery
