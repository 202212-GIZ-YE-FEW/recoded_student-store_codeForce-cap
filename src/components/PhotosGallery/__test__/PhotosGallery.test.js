import renderer from "react-test-renderer"

import PhotosGallery from ".."

it("renders correctly", () => {
  const tree = renderer.create(<PhotosGallery />).toJSON()
  expect(tree).toMatchSnapshot()
})
