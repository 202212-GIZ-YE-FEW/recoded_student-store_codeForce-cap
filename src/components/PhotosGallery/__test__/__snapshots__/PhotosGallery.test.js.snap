// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="max-w-[1250px] mx-auto my-[50px]"
>
  <div />
</section>
`;
